cli/wallet/multisig.go

```diff
@@ -89,3 +94,14 @@ func signStoredTransaction(ctx *cli.Context) error {
 	fmt.Fprintln(ctx.App.Writer, tx.Hash().StringLE())
 	return nil
 }
+
+func dumpTransaction(ctx *cli.Context, tx *transaction.Transaction) {
```

Comment:
Maybe we can reuse `query tx` code here and show signers as well (dumping script opcodes is also nice to have for `query tx`)?

Reply:
TBH it won't be a pleasant user experience e.g. for deploy transactions where script can easily be longer than 1k lines. But this is only showed with `--verbose`, so we are probably ok with this.